MdePkg/MdePkg.dec

```diff
@@ -1893,6 +1893,9 @@
   ## Include/Protocol/EraseBlock.h
   gEfiEraseBlockProtocolGuid           = { 0x95a9a93e, 0xa86e, 0x4926, {0xaa, 0xef, 0x99, 0x18, 0xe7, 0x72, 0xd9, 0x87 }}
 
+  ## Include/Protocol/MediaSanitize.h
+  gEfiMediaSanitizeProtocolGuid        = { 0x0d799a99, 0x25af, 0x429e, {0x92, 0x72, 0xd0, 0xb2, 0x7d, 0x6d, 0x5f, 0x14 }}

```

Comment:
There are lots of protocols defined in the code tree.  
Some are UEFI defined. 
Some are Microsoft defined in specs on what was MSDN
Some are edk2 defined for use within Edk2 based projects
Some are defined by Project Mu
Some are defined by silicon partners for a given silicon implementation.  .......
you get the idea.  

So it just depends on what you business case is and what group you want to work with to get it defined.  Obviously Edk2/Mu have less work.  UEFI isn't that difficult either given you have buttoned up your use case and are willing to spend a few hours for a few weeks on the meetings.  

Since here you are adding it to mu_basecore/MdePkg and you are using Efi prefix you are conveying that you are targeting UEFI spec.  If you were to remove Efi prefix and put it in MdeModulePkg you could target edkII or mu depending on prefix.

Reply:
I went ahead and moved the Media Sanitize protocol from MdePkg to MdeModulePkg (removing the "EFI_" prefix). Please look over my changes to ensure I did this properly. Thanks.